composer.json

```diff
@@ -4,26 +4,29 @@
     "description": "",
     "license": "MIT",
     "require": {
-        "php": "^8.0",
+        "php": "^8.1",
         "ext-json": "*",
-        "monolog/monolog": "^1.24 || ^2.0",
-        "symfony/config": "^3.4 || ^4.0 || ^5.0 || ^6.0",
-        "symfony/console": "^3.4 || ^4.0 || ^5.0 || ^6.0",
-        "symfony/dependency-injection": "^3.4 || ^4.0 || ^5.0 || ^6.0",
-        "symfony/http-kernel": "^3.4 || ^4.0 || ^5.0 || ^6.0",
+        "monolog/monolog": "^2.0",
+        "symfony/config": "6.0.*",
+        "symfony/console": "6.0.*",
+        "symfony/dependency-injection": "6.0.*",
+        "symfony/http-kernel": "6.0.*",
```

Comment:
we use `6.0.*` in our microservices to be strict
but for bundles we can keep `^6.0` like before I think